### PR TITLE
Search on homepage

### PR DESCRIPTION
**Pull request by mariamicioni**

### CODE REVIEW

File: app/controllers/bears_controller.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 class BearsController < ApplicationController
+  before_action :find_bear, only: [:edit, :update, :destroy, :show]
   def index
-    @bears = Bear.all
+      @bears = Bear.all
```
Comment (natebans) [inline app/controllers/bears_controller.rb:4]: this line is indented too far

File: app/controllers/bears_controller.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
   end
 
+
```
Comment (natebans) [inline app/controllers/bears_controller.rb:7]: extra line can be removed

File: app/controllers/pages_controller.rb [hunk 3]
```diff
@@ -2,7 +2,10 @@ class PagesController < ApplicationController
   skip_before_action :authenticate_user!, only: [:home]
 
   def home
-    @bears = Bear.all
+      if params[:name].present?
+        @bears = Bear.search_name(params[:name])
+      else
+        @bears = Bear.all
+      end
```
Comment (natebans) [inline app/controllers/pages_controller.rb:9]: it looks like the if statement is indented too far

**Review by natebans — CHANGES_REQUESTED** [review]

File: app/controllers/bears_controller.rb (hunk 1 above)
Comment (mariamicioni) [reply app/controllers/bears_controller.rb]: fixed in next commit

File: app/controllers/pages_controller.rb (hunk 3 above)
Comment (mariamicioni) [reply app/controllers/pages_controller.rb]: fixed in next commit

File: app/controllers/bears_controller.rb (hunk 2 above)
Comment (mariamicioni) [reply app/controllers/bears_controller.rb]: removed in next commit

**Comment by natebans** [comment]
looks good!

**Pull request merged**